wrangler-core/src/main/java/io/cdap/wrangler/utils/ColumnConverter.java

```diff
@@ -210,25 +213,30 @@ private static Object convertType(String col, String toType, Object object) thro
       }
 
       case "DECIMAL": {
+        BigDecimal decimal = null;
```

Comment:
nit: Do you want to call the variable something else? like convertedObject or something else instead of 'decimal'?